### ISSUE TITLE
Documentation template as default instead of minima? 

### ISSUE THREAD

**Issue by Jiraiyah** [jekyll/jekyll#7676]
Hi
I was wondering if there is a chance we would get two templates as default integration when we install jekyll?

The first would be minima to keep everything simple and let developers start from bare minimum if they need to.

But, for people like me who don't know anything about css and template generation, how about giving us a chance to just change the minima name in the config yaml file to something else and get the exact carbon copy of the template you have on the website?

I tried porting the documentation repository and work with it. Encountered two problems that i didn't know how to fix :

1- Links were not working on github : 
On local, any page link was fine. but on github, the documentation repository won't have anything for localurl, so, any link you have for pages etc, won't respect the repository and would try to read things from main subdomain.

2- CSS was not working : 
Again, on local everything would look correct. But when you would try getting it on github pages, you would see a white background, no css and no formatting at all.

Long story short, I believe there are many people like me who have 0 knowledge about web styling and development but the moment they see your website's template they wish they had something like that. Giving us an easy way and chance to have it, would not only satisfy the hearts of many, but I believe at the same time, would rise the number of users who would try and love jekyll daily based. Because even though I don't know how to style, when i have the base working, I can easily put time and effort to change colors and font and font sizes and stuff like that, in other words, using the base jekyll's own website template, and then changing the look of it to get my own taste.

I would really appreciate such a thing and I believe many people would appreciate it.

Thanks

**Comment by DirtyF** [comment]
Please use a documentation (gem-based) theme.
You can find some here: https://jekyllthemes.io/jekyll-documentation-themes
